backend/src/validators/user.ts

```diff
@@ -0,0 +1,9 @@
+import { ValidationChain, body } from "express-validator";
+
+export const createUser: ValidationChain[] = [
+  body("name").notEmpty().isString(),
+  body("accountType").notEmpty().isIn(["admin", "team"]),
+  body("approvalStatus").notEmpty().isBoolean(),
```

Comment:
Again lets remove this from the req.body and in the controller we will set this to false before storing on Mongodb.